docs/requirements.md

```diff
@@ -1,51 +1,140 @@
+#### Transactions
+{:.no_toc}
+ScalarDB requires each underlying database to provide at least the following capabilities for running transactions on the databases:
+- Linearizable read and conditional mutations (write and delete) on a single database record.
+- Durability of written database records.
+- Ability to store arbitrary data besides application data in each database record.
 
-## Cassandra or Cassandra-compatible database requirements
+{% capture notice--info %}
+**Note**
+
+You need to have database accounts that have enough privileges to access the databases thorough ScalarDB since ScalarDB runs on the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. ScalarDB basically requires a fully privileged account to access the underlying databases.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+#### Analytics
+{:.no_toc}
+ScalarDB requires each underlying database to provide the following capability for running analytics on the databases:
+- Ability to return only committed records.
+
+### How to configure databases to achieve the general requirements
+
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'JDBC_databases', 'tabset-1')" id="defaultOpen-1">JDBC databases</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB', 'tabset-1')">DynamoDB</button>
+  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL', 'tabset-1')">Cosmos DB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'Cassandra', 'tabset-1')">Cassandra</button>
+</div>
+
+<div id="JDBC_databases" class="tabcontent" markdown="1">
+
+#### Transactions
+{:.no_toc}
+- Use a single primary server or synchronized multi-primary servers for all operations. (No reads on read replicas that are asynchronously replicated from a primary database.)
+- Use read committed or stricter isolation levels.
+
+#### Analytics
+{:.no_toc}
+- Use read committed or stricter isolation levels.
 
-The following are requirements to make ScalarDB on Cassandra or Cassandra-compatible databases work properly and for storage operations with `LINEARIZABLE` to provide linearizability and for transaction operations with `SERIALIZABLE` to provide strict serializability.
+</div>
 
-### Ensure durability in Cassandra
+<div id="DynamoDB" class="tabcontent" markdown="1">
 
-In **cassandra.yaml**, you must change `commitlog_sync` from the default `periodic` to `batch` or `group` to ensure durability in Cassandra.
+#### Transactions
+{:.no_toc}
+- Use a single primary region for all operations. (No reads and writes on global tables in non-primary regions.)
+  - There is no concept for primary regions in DynamoDB, so you must designate a primary region by yourself.
```

Comment:
```suggestion
  - There is no concept for primary regions in DynamoDB global tables, so you must designate a primary region by yourself.
```
I think the concept of multi-region only exists in global tables.

Reply:
@komamitsu Thanks.
It doesn't really talk about global tables, let me revise the whole sentence.
How about this?

```
A primary region is not a concept in DynamoDB, even when you use multiple regions with DynamoDB global tables, so you must designate a primary region on your own.
```
@josh-wong

Reply:
@feeblefakie 💯